app/controllers/idt/api/v2/distributions_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class Idt::Api::V2::DistributionsController < Idt::Api::V1::BaseController
+  protect_from_forgery with: :exception
+  before_action :verify_access
+
+  # rubocop:disable Metrics/MethodLength, Naming/AccessorMethodName
+  def get_distribution
+    # rubocop:enable Metrics/MethodLength, Naming/AccessorMethodName
+    distribution_id = params[:distribution_id]
+    # Checks if the distribution id is blank and if it exists with the database
+    if distribution_id.blank? || !valid_id?(distribution_id)
+      render_error(400, "Distribution Does Not Exist Or Id is blank", distribution_id)
+      return
+    end
+
+    begin
+      # Retrieves the distribution package from the PacMan API
+      distribution = PacManService.get_distribution_request(distribution_id)
+      response_code = distribution.code
+      if response_code != 200
+        fail StandardError
+      end
+      # Handles errors when making any requests both from Pacman and the DB
+    rescue StandardError
+      case response_code
+      when 404
+        render_error(response_code, { "id": distribution_id, "status": "PENDING_ESTABLISHMENT" }, distribution_id)
```

Comment:
```suggestion
        render json: { id: distribution_id, status: "PENDING_ESTABLISHMENT" }
```
I think for now we can treat this as not necessarily an error.